### DISCUSSION TITLE
Z axis not staying consistent

### DISCUSSION THREAD

**Discussion by dizzlej515** [blackfyre/flsun.community#99] [Q&A]

**Comment by dizzlej515** [comment]
Print may start okay but as print continues it tends to almost nose dive further and further. Making me either sit there and watch it and continue to raise the Z and hope for the best or just say screw it and be upset about a $500 printer

**Comment by MarBel78** [comment]
Like this?
![image](https://user-images.githubusercontent.com/12729273/206489775-3703e922-c2da-47be-b4ab-efe0882875a9.png)
